amplitude/test/testPeakShapeInstableIsobar.cc

```diff
@@ -272,7 +272,7 @@ class threeBodyDynAmpMc : public threeBodyDynAmpInt {
 
 public:
 
-	threeBodyDynAmpMc(const dynamicAmplitude& isobarAmp,
+	threeBodyDynAmpMc(const dynamicAmplitude&,
 	                  const double            mass         = 0,
 	                  const double            width        = 0,
 	                  const unsigned int      L            = 0,
```

Comment:
parameter should not be removed, but added to the call of the constructor of the base class (I wonder why this ever compiled, probably because it was not used anywhere and optimised away)